lib/analyzer/types.d.ts

```diff
@@ -9,6 +9,12 @@ export interface AnalyzerPkg {
   AutoInstalled?: boolean;
 }
 
+export interface AnalyzerResult {
+  Image: string;
+  AnalyzeType: string;
+  Analysis: AnalyzerPkg[];
```

Comment:
I know names `AnalyzeType` and `Analysis` were here before, but could we improve them now, as we are adding more implementations of this interface?

It really confused me when reading the code. Perhaps we could name them `pkgManager` and `packages`, as they're already called elsewhere?

Reply:
As noted on Slack, I started doing this but it turned out bigger than expected. I am happy to do this in a separate PR!